Dockerfile.template

```diff
@@ -0,0 +1,12 @@
+FROM {{ from }}
+ENV NPM_CONFIG_LOGLEVEL warn
+
+WORKDIR /opt/cs-service/
+
+{{#extraInstructions}}
```

Comment:
The extra blank line in the output bothers me slightly. Would it go away if you do these in one line?

```{{#extraInstructions}} {{{.}}} {{/extraInstructions}}```